camera_info_manager/src/camera_info_manager.cpp

```diff
@@ -313,6 +316,23 @@ bool CameraInfoManager::loadCameraInfo(const std::string & url)
   return loadCalibration(url, cname);
 }
 
+std::string get_env(const char * env_var)
+{
+#ifndef _WIN32
+  auto env_variable = getenv(env_var);
+#else
+  char * output_env_var = nullptr;
+  size_t env_var_size;
+  _dupenv_s(&output_env_var, &env_var_size, env_var);
+
+  std::string env_variable = "";
+  if (output_env_var) {
+    env_variable = std::string(output_env_var);
+    free(output_env_var);
+  }
+#endif
+  return env_variable;
+}
```

Comment:
Can I suggest using `rcutils_get_env` here instead?  Especially since this is new code.

Reply:
I'm going to use the `rcpputils` variant, since we already have that dependency.